Terminal.Gui/Application/Application.Mouse.cs

```diff
@@ -147,7 +150,7 @@ internal static void RaiseMouseEvent (MouseEventArgs mouseEvent)
         //Debug.Assert (mouseEvent.Position == mouseEvent.ScreenPosition);
         mouseEvent.Position = mouseEvent.ScreenPosition;
 
-        List<View?> currentViewsUnderMouse = View.GetViewsUnderMouse (mouseEvent.ScreenPosition);
+        List<View?> currentViewsUnderMouse = View.GetViewsUnderLocation (mouseEvent.ScreenPosition, ViewportSettings.TransparentMouse);
```

Comment:
Nice, this is a more professional name for the method 👍 

We could change Location to 'ScreenPoint' or 'Point' to be even more specific (depending on what this function uses for its coordinate space)

Reply:
Good call

Now: 


```cs
    /// <summary>
    ///     Gets the Views that are under <paramref name="screenLocation"/>, including Adornments. The list is ordered by depth. The
    ///     deepest
    ///     View is at the end of the list (the top most View is at element 0).
    /// </summary>
    /// <param name="screenLocation">Screen-relative location.</param>
    /// <param name="excludeViewportSettingsFlags">
    ///     If set, excludes Views that have the <see cref="ViewportSettings.Transparent"/> or <see cref="ViewportSettings.TransparentMouse"/>
    ///     flags set in their ViewportSettings.
    /// </param>
    public static List<View?> GetViewsUnderLocation (in Point screenLocation, ViewportSettings excludeViewportSettingsFlags)
```